README.md

```diff
@@ -10,6 +10,53 @@ Install via npm:
 $ npm i @godaddy/split-data-loader --save
 ```
 
+## Usage
+
+Use this package if you are using Split.io for experimentation on a webpage and want to take advantage of browser caching with [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
+
+Please note that this package is intended to be used in tandem with a split data serializer to create the `serializedData` parameter properly.
+An example of such serializer is [`split-node-serializer`](https://github.com/godaddy/split-node-serializer).
+
+```js
+import loadDataIntoLocalStorage from '@godaddy/split-data-loader'
+
+loadDataIntoLocalStorage({
+  serializedData: {
+    segmentsData: {
+      "segment_1": ['test-visitor-1', 'test-visitor-2', 'test-shopper-1'],
+      "segment_2": ['test-visitor-1', 'test-visitor-3', 'test-shopper-2']
+    },
+    since: '-1',
+    splitsData: {
+      "experiment_1": "{ name: 'experiment_1', status: 'foo' }",
+      "experiment_2": "{ name: 'experiment_2', status: 'bar' }"
+    },
+    usingSegmentsCount: 2
+  },
+  userId: 'test-visitor-1'
+})
+```
+
+### Function
+
+#### loadDataIntoLocalStorage
+
+You can use this function to store the following in browser `localStorage`:
+- Freshness of your data
+- Serialized split data
+- Count of splits using segments
+- Indicators for which segments a `userId` is a part of
```

Comment:
i'm still confused about this, what's `userId` here? the segment data that the serializer returns is not specific to an individual GoDaddy customer. it's just segments in use by splits and the keys (the values in `added` that are part of those segments)

Reply:
i'd just call this `serialized segment data`

Reply:
The `userId` is either the hashed shopperId or visitorGuid value that pertains to the shopper/visitor. The `userId` is the "key" that would be a value in the `added` array.

Reply:
whoops, i was misreading this. how it is makes sense to me